package.json

```diff
@@ -92,13 +92,14 @@
     "multer": "^1.3.0",
     "node-html-parser": "^2.0.0",
     "q": "^1.4.1",
+    "re2": "^1.17.4",
     "request": "^2.69.0",
     "serve-favicon": "^2.5.0",
-    "steal": "^1.6.3",
+    "steal": "^2.2.4",
     "steal-css": "^1.3.1",
     "through2": "^2.0.1",
     "typeahead.js": "git://github.com/twitter/typeahead.js.git#v0.11.1",
-    "url-regex-safe": "^2.0.2",
+    "url-regex-safe": "^3.0.0",
```

Comment:
i've had an issue where changing url-regex-safe broke a default behavior for testing `isAbsolute` - I think there is a test for that, but we should maybe double check it's still working https://github.com/CCALI/a2jdat/blob/02336bea9a4e645ada8f56c8d7a553ba0d539cbd/src/util/paths.js#L43

Reply:
Looking again, the change was likely not solely responsible for breaking default behavior. It was likely due to path construction that now may require VIEWER_PATH and harder resrictions on GUIDES_URL. Though this is a major version change, it looks like what changes is the dependency of re2 and theoretically not any actual behavior. Testing on standalone and preview, with new and compensating settings, this still seems to work. I'm not sure how else to test this

Reply:
Found a case where this is failing: on dev.a2j.org, since the folder ends in `.zip`, despite leading with `..` making it impossible to be a URL it is detected as a URL

Reply:
looks like this is fixed by adding `{strict:false}` to URLregex

Reply:
👍